scripts/ynmp/update.py

```diff
@@ -71,7 +71,7 @@ def update_from(csv_url, data):
             'name': name,
             'on_behalf_of_id': data['orgs'][party],
             'person_id': person_id,
-            'start_date': '2015-05-08',
+            'start_date': '2017-06-08',
```

Comment:
A tiny point, but the `start_date` was previously the day after the election (and ISTR parlparse does this consistently) so I'd make this the 9th rather than the 8th.